From cf4f428e335752f750829ae4408a4cf951b878f7 Mon Sep 17 00:00:00 2001
From: dhacker29 <dhackerdvm@gmail.com>
Date: Sun, 6 Dec 2015 22:53:52 -0500
Subject: [PATCH 02/15] sepolicy: Set the context for fsck.exfat/ntfs to
 fsck_exec

This matches the policy for fsck.f2fs, although it still needs to run
as fsck_untrusted for public volumes

Change-Id: Ia04e7f8902e53a9926a87f0c99e603611cc39c5d
---
 common/private/file_contexts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/common/private/file_contexts b/common/private/file_contexts
index 8d6364f..bf971f9 100644
--- a/common/private/file_contexts
+++ b/common/private/file_contexts
@@ -1,3 +1,7 @@
+# Filesystem tools
+/system/bin/fsck\.exfat                 u:object_r:fsck_exec:s0
+/system/bin/fsck\.ntfs                  u:object_r:fsck_exec:s0
+
 # OTA packages
 /data/lineageos_updates(/.*)?           u:object_r:ota_package_file:s0
 
-- 
2.7.4

